redisinsight/api/src/modules/database/providers/database-overview.provider.ts

```diff
@@ -291,6 +298,73 @@ export class DatabaseOverviewProvider {
     return nodesTotal.reduce((prev, cur) => prev + cur, 0);
   }
 
+  /**
+   * Calculates maximum CPU percentage based on number of nodes/shards
+   * For clusters: max = number of primary nodes * 100% (always returned, no cap)
+   * For standalone: detect I/O threads via CONFIG GET (no estimation)
+   *
+   * Example of calculation:
+   * 2 shards: 2 * 100 = 200%
+   * 4 threads: 4 * 100 = 400%
+   */
+  private async calculateMaxCpuPercentage(
+    client: RedisClient,
+  ): Promise<number | undefined> {
+    // For cluster, return the number of primary nodes * 100%
+    if (client.getConnectionType() === RedisClientConnectionType.CLUSTER) {
```

Comment:
what about standalone? I thought that this method should process both connections

Reply:
Yes, it's handled below:
- check if we deal with a cluster and count its nodes
- otherwise, treat it as a standalone and check whether we have multiple threads within